shared/p2p/options_test.go

```diff
@@ -1,11 +1,50 @@
 package p2p
 
 import (
+	"io/ioutil"
+	"os"
 	"testing"
+
+	crypto "github.com/libp2p/go-libp2p-crypto"
+	"github.com/libp2p/go-libp2p/config"
 )
 
 func TestBuildOptions(t *testing.T) {
 	opts := buildOptions(&ServerConfig{})
 
 	_ = opts
 }
+
+func TestPrivateKeyLoading(t *testing.T) {
+	file, err := ioutil.TempFile("", "key")
```

Comment:
Please use the tmp dir from the testutil: `testutil.TempDir()` github.com/prysmaticlabs/prysm/shared/testutil.